docs: Revamp API Platform Admin docs (#2141)
api-platform/docs: admin/schema-org.md

> # Customizing the Schema

Reviewer: Maybe the filename should just be `schema.md`.

Author: I agree. I renamed the file to workaround a bug causing the section to not be highlighted in the left menu, but we can go even further with the renaming. I'll push a new commit.

Both [`HydraAdmin`](./components.md#hydraadmin) and [`OpenApiAdmin`](./components.md#openapiadmin) leverage introspection of the API schema to discover its capabilities, like **filtering** and **sorting**.

They also detect wether the API has real-time capabilities using [Mercure](./real-time-mercure.md), and automatically enable it if it does.

Lastly, API Platform Admin has native support for the popular [Schema.org](./schema-org.md#about-schemaorg) vocabulary, which enables it to automatically use the field type matching your data, or display a related resource's name instead of its IRI.

## Adding Filtering Capabilities

> You can add the [`ApiFilter` attribute](../core/filters.md#apifilter-attribute) to an Entity to configure a filter on a property.

Reviewer: ```suggestion
You can add the [`ApiFilter` attribute](../core/filters.md#apifilter-attribute) to an API Platform resource to configure a filter on a property.
```

> For instance, here is how configure filtering on the `id`, `title` and `author` properties of a `Book` entity:

Reviewer: ```suggestion
For instance, here is how configure filtering on the `id`, `title` and `author` properties of a `Book` resource:
```

```php
<?php
// api/src/Entity/Book.php
namespace App\Entity;

use ApiPlatform\Metadata\ApiFilter;
use ApiPlatform\Metadata\ApiResource;
use ApiPlatform\Doctrine\Orm\Filter\SearchFilter;
use Doctrine\ORM\Mapping as ORM;

#[ApiResource]
#[ORM\Entity]
#[ApiFilter(SearchFilter::class, properties: [
    'id' => 'exact', 
    'title' => 'ipartial', 
    'author' => 'ipartial'
])]
class Book
{
    // ...
}
```

If you are using the guessers, the Admin will automatically update the Book list view to include a filter on the selected properties.

![Filtering on the title property](./images/admin-filter.png)

**Tip:** Learn more about the [`ApiFilter` attribute](../core/filters.md#apifilter-attribute) in the core documentation.

## Adding Sorting Capabilities

You can also use the [`ApiFilter` attribute](../core/filters.md#apifilter-attribute) on an Entity to configure sorting.

> For instance, here is how to configure sorting on the `id`, `isbn`, `title`, `author` and `publicationDate` properties of a `Book` entity:

Reviewer: ```suggestion
For instance, here is how to configure sorting on the `id`, `isbn`, `title`, `author` and `publicationDate` properties of a `Book` resource:
```
